CMakeLists.txt

```diff
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.13)
-cmake_policy(SET CMP0077 NEW)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Remember to remove the changes to CMakeLists.txt from this PR, as you already made them in a separate commit.

Reply:
Yup, done